### DISCUSSION TITLE
KDL template (accepts variables)

### DISCUSSION THREAD

**Discussion by morenohernan** [zellij-org/zellij#2773] [Q&A] [answered]
Hi, 

is it possible for Zellij's kdl to accept variables to receive inputs from terminal so we can avoid hardcoded paths ,etc .. ?
<img width="1074" alt="image" src="https://github.com/zellij-org/zellij/assets/700619/a6279f85-bdef-4035-a2c9-cb8d294ff31f">

in this way the kdl is like a template to be populated and Zellij renders accordingly..

**Comment by Lockszmith-GH** [comment, marked as answer]
I asked a similar question a while back, the answer was 'not yet', and [I posted how I solved it](https://github.com/zellij-org/zellij/discussions/2142#discussioncomment-4909963) for now.

**Reply by morenohernan** [reply]
Thanks @Lockszmith-GH , it is helpful your approach for my stuff..
